docs/content/upgrade-guides/plugins/meilisearch/1-0-0.md

```diff
@@ -0,0 +1,72 @@
+# Meilisearch Search: v1.0.0
+
+Version 1.0.0 of our official Meilisearch plugin comes with breaking changes to the plugin options that are passed to the Meilisearch service through `medusa-config.js`.
+
+## Overview
+
+In the new version of the Meilisearch search plugin, two new plugin configuration properties are introduced; `transformer` and `primaryKey`.  As a result, the way indexes in Meilisearch are configured has changed. 
+
+The new object shape for the plugin options is as follows:
+
+```jsx
+{
+  config: {
+		host: string
+		apiKey: string
+	}
+  settings: {
+    [indexName]: {
+      indexSettings: {
+        searchableAttributes: string[]
+        displayedAttributes: string[]
+				...
+      }
+      transformer?: <T>(medusaEntity: T) => unknown
+    }
+  }
+}
+```
```

Comment:
As we have ESLint configured on languages like `js`, `jsx`, `ts`, and `tsx`, this block produces errors related to JavaScript. I suggest that we change the block to the following:

~~~suggestion
```js title=medusa-config.js
const plugins = [
  // ...
  {
    config: {
      host,
      apiKey
    }
    settings: {
      indexName: {
        indexSettings: {
          searchableAttributes,
          displayedAttributes
          // ...
        }
        transformer?: (product) => ({
          id: product.id,
          // ...
        })
      }
    }
  }
]
```
~~~

And then we can link to the MeiliSearch plugin for more details on what each of these mean, although I don't think that's necessary considering you already cover it later in the upgrade guide.